lib/engine/round/base.rb

```diff
@@ -125,6 +125,10 @@ def active_step(entity = nil)
         @active_step ||= @steps.find { |step| step.active? && step.blocking? }
       end
 
+      def potential_blocking_step
```

Comment:
I'm not super happy with this name, I didn't want it to be `blocking_step` since it doesn't actually call `step.blocking?`. Maybe if I move it into Game (see https://github.com/tobymao/18xx/pull/2870#discussion_r549816469) I could rename it to `ability_blocking_step` or `ability_current_step` or something? That groups it with the other `ability_` methods and makes it a little more clear this is really only for the special case of looking at abilities.